project/Build.scala

```diff
@@ -38,7 +38,7 @@ object SlickBuild extends Build {
     val testDBs = Seq(
       h2,
       "org.xerial" % "sqlite-jdbc" % "3.8.7",
-      "org.apache.derby" % "derby" % "10.9.1.0",
+      "org.apache.derby" % "derby" % "10.10.1.1",
```

Comment:
I've bump the version because I was getting trouble with an order by for derby, seems to be fixed in this version